Source/WebCore/Modules/WebGPU/GPUSwapChainDescriptor.h

```diff
@@ -39,17 +41,25 @@ struct GPUSwapChainDescriptor : public GPUObjectDescriptorBase {
     {
         return {
             { label },
-            WebCore::convertToBacking(size),
-            sampleCount,
             WebCore::convertToBacking(format),
             convertTextureUsageFlagsToBacking(usage),
+            viewFormats.map([] (auto& viewFormat) {
+                return WebCore::convertToBacking(viewFormat);
+            }),
+            WebCore::convertToBacking(colorSpace),
+            WebCore::convertToBacking(compositingAlphaMode),
+            width,
+            height,
         };
     }
 
-    GPUExtent3D size;
-    GPUSize32 sampleCount { 1 };
     GPUTextureFormat format { GPUTextureFormat::R8unorm };
```

Comment:
should we make this `GPUTextureFormat::BGR8unorm` while we are here or does it not really matter since its a required field from the web api?

Reply:
I don't think it really matters, but I'm happy to change it now.